apps/website/.vuepress/public/core-usage-demos/toggle/basic.html

```diff
@@ -0,0 +1,4 @@
+<cds-toggle>
+  <label>toggle</label>
+  <input type="checkbox" checked />
+</cds-toggle>
```

Comment:
Why do we need Clarity City loaded here manually?

Reply:
@mathisscott noticed that even though the core styles were loading, the font wasn't. This is the work around to get it loaded for now.